packages/manager/modules/pci/src/projects/project/failover-ips/order/order/order.controller.js

```diff
@@ -59,4 +60,8 @@ export default class FailoverIpController {
     this.$window.open(`${ORDER_URL}${JSURL.stringify([order])}`, '_blank');
     this.goBack();
   }
+
+  static isDisabledInstance(instance) {
```

Comment:
You are repeating this function over and over, maybe you should make a service :)

Reply:
Even better, you should maybe add it to the `Instance` model :)

Reply:
Wouldn't the instance do too much though ? 🤔

Reply:
The `instance` may contain its capabilities or have an `InstanceCapabilities` service with a `getForInstance(instance)` method that returns an `InstanceCapabilities` object.

Reply:
A new method `isCapabilityEnabled` has been added to the `Instance` class, since it is very closely related to an `Instance`. This also avoids the code complication of injecting a new service every required and assigning it to scope, if the `capabilities` is a part of a service.
Please let me know your thoughts if you think there is a better approach :)